ntfs_fr.md

```diff
@@ -528,9 +529,24 @@ Colonne | Type | Contrainte | Commentaire
 address_id | chaine | Requis | Identifiant de l'adresse
 street_name | chaine | Requis | Nom de la voierie
 house_number | chaine | Optionnel | Numéro du seuil
+admin_level_8_id | chaine | Optionnel | Identifiant de l'administration de niveau 8 dans le fichier `administrative_regions.txt`
+admin_level_9_id | chaine | Optionnel | Identifiant de l'administration de niveau 9 dans le fichier `administrative_regions.txt`
+admin_level_10_id | chaine | Optionnel | Identifiant de l'administration de niveau 10 dans le fichier `administrative_regions.txt`
+
+### administration_regions.txt (optionnel)
+Colonne | Type | Contrainte | Commentaire
+--- | --- | --- | ---
+admin_id | chaine | Requis | Identifiant de la région administrative
+admin_insee | chaine | Requis | Code INSEE
+admin_level | entier | Requis | Niveau de la région administrative
+admin_name | chaine | Requis | Nom de la région administrative
+admin_label | chaine | Requis | Label de la région administrative
```

Comment:
Is it necessary ? If label = name + (insee) it could be computed

Reply:
I keep label if later it will evolve

Reply:
yes, I think you can keep both, label is not always name + (insee)
![image](https://github.com/hove-io/ntfs-specification/assets/52410095/c2951cf1-ccf2-4993-9636-b70e347bb5f3)